docs/interactions.rst

```diff
@@ -0,0 +1,126 @@
+:orphan:
+
+.. currentmodule:: nextcord
+
+.. _interactions:
+   
+Slash Commands
+==============
+
+Since Discord has added interaction commands a feature with many possibilities, we at Nextcord have decided to add them to our fleet of features.
```

Comment:
Not a fan of this line, it feels *too* 'friendly'/'joky'. It's expected of us as a lib to implement Discord's API feature's, but this sounds as if it was just a general feeling that we should

Reply:
Yeah I can’t find any good replacement to this line

Reply:
Would it be better to just not include any description here?